iexec-sms-library/src/main/java/com/iexec/sms/api/config/GramineServicesProperties.java

```diff
@@ -0,0 +1,19 @@
+package com.iexec.sms.api.config;
+
+import com.iexec.common.tee.TeeEnclaveProvider;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class GramineServicesProperties extends TeeServicesProperties {
+
+    public GramineServicesProperties() {
```

Comment:
When do we need this constructor?

Reply:
`GramineServicesProperties` and `SconeServicesProperties` are exchanged through network. They should be able to be serialized and deserialized. The second action requires a default constructor, as far as I remember.